src/main/java/net/snowflake/client/core/SFJsonResultSet.java

```diff
@@ -233,7 +233,7 @@ public byte[] getBytes(int columnIndex) throws SFException {
   }
 
   public Date getDate(int columnIndex) throws SFException {
-    return getDate(columnIndex, TimeZone.getDefault());
+    return getDate(columnIndex, TimeZone.getTimeZone("UTC"));
```

Comment:
Let's add tests showing different case when called getString, getDate, getObject on the same column with different timezones in the session and in the jvm for json and for arrow result sets